dts/bindings/gpio/it8xxx2-gpio.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: This binding gives a base representation of the ITE gpio
+
+compatible: "it8xxx2-gpio"
```

Comment:
@galak This is a new SoC family. Should we require that all compatible properties are prefixed by `ite,`? As in `ite,it8xxx2-gpio` in case of GPIO node.

Reply:
Yes, @galak has reminded me to revised this :) . I'll update this at the next commit.